jetbrains-core/src/software/aws/toolkits/jetbrains/services/codemodernizer/model/CodeModernizerSessionContext.kt

```diff
@@ -225,6 +242,84 @@ data class CodeModernizerSessionContext(
             }
         }
 
+        // 3. intellij-bundled maven runner
```

Comment:
needs to guard for presence of maven plugin, or you'll get some variant of `NoClassDefFoundError` at runtime

Reply:
BLOCKER: Sounds like a blocker. Is this `if (shouldTryMvnCommand)` enough?

Reply:
We have 
```
try {
     transformMvnRunner.run(params, mvnsettings, createdDependencies)
} catch (t: Throwable) {
...
}
```
If there is no maven plugin, it would throw the exception and we can catch it during the maven progress. Is this enough?

Reply:
Thanks so much for pointing out this issue!! I moved the `val params = MavenRunnerParameters()`block to the try catch so that we can catch the `NoClassDefFoundError` when customers do not enable the bundled maven.